lib/swagger/diff/diff.rb

```diff
@@ -112,24 +174,46 @@ def incompatible_response_attributes
         ret
       end
 
-      def incompatible_response_attributes_enumerator
+      def new_or_changed_response_attributes
```

Comment:
The enumerators need to be defined as they are (two are used multiple times, while two are only used once), but I DRYed up the hash population.